sql/setup/set_meta.sh

```diff
@@ -16,12 +16,11 @@ if [ "$#" -ne 2 ] ; then
     exit 1
 fi
 
-NODENAME=$1
```

Comment:
Because I could find no usage of this script that passed more than one argument, and that argument was a hostname. The underlying SQL function `set_meta`, called by this shell script, accepts no argument that resembles a "nodename".

This change was part of an unsuccessful attempt to get the "unit tests" in `test/unit` working locally.